internal/config/namespace.go

```diff
@@ -31,8 +34,22 @@ var (
 	// this list is removed from all managed namespaces unless specifically specified by the HC of the
 	// namespace or one of its ancestors.
 	managedAnnotations []*regexp.Regexp
+
+	// hncNamespace is the namespace where hnc-manager and hnc resources deployed. It set by commandline argument,
+	// default to hnc-system.
+	hncNamespace string
 )
 
+var (
+	once sync.Once
+)
+
+func SetHncNamespace(ns string) {
```

Comment:
Convention is `SetHNCNamespace` (acronyms are capitalized the same way) - https://blog.devgenius.io/write-better-go-code-by-using-consistent-case-for-initials-in-variables-b84e862fd0ba